### PR TITLE
TogglePill radio keyboard nav

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/WC-191

#### Description
Radio-style `TogglePill`s are now keyboard navigable. I built a wrapper component as a part of this fix, and I created a [follow-up ticket](https://meetup.atlassian.net/browse/SDS-589) to replace `withToggleControl` usage

#### Screenshots (if applicable)
radio_keyboardnav

### CODE REVIEW

File: src/utils/components/ToggleWrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+	render() {
+		const isInput = this.props.type == 'radio' || this.props.type == 'checkbox';
+
+		return (
+			<ConditionalWrap
+				condition={!isInput}
+				wrap={children => (
+					<div
+						role="button"
```
Comment (akdetrick) [inline src/utils/components/ToggleWrapper.jsx]: Do we always want to set this role, or only if `!isInput`?

Comment (mperrotti) [reply src/utils/components/ToggleWrapper.jsx]: Only if `!isInput`

File: src/utils/components/ToggleWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		}
+	}
+
+	onKeyUp(e) {
+		const isActivatingButton = [
+			' ', /* space bar */
+			'Enter'
+		].some(key => e.key === key);
+
+		if (isActivatingButton) { // && this.props.type !== 'radio'
```
[the remark this answers is not in the document]
Comment (mperrotti) [reply src/utils/components/ToggleWrapper.jsx]: This comment is not intentional and should have been removed 🙃

File: src/utils/components/ToggleWrapper.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+		this.onKeyUp = this.onKeyUp.bind(this);
+	}
+
+	componentWillReceiveProps(nextProps) {
+		if (nextProps.isActive !== this.props.isActive) {
+			this.toggleActive();
+		}
+	}
+
+	toggleBool(e) {
```
[the remark this answers is not in the document]
Comment (mperrotti) [reply src/utils/components/ToggleWrapper.jsx]: There was a reason when I had this code in `withToggleControl`, where this code comes from, but I can't remember what it was

**Pull request merged**